proto/composerapis/astria/composer/v1alpha1/grpc_collector.proto

```diff
@@ -0,0 +1,18 @@
+syntax = 'proto3';
+
+package astria.composer.v1alpha1;
+
+import "google/protobuf/empty.proto";
+
+message RollupTx {
+  string rollup_id = 1;
```

Comment:
Should this be a generic `string` or should we also require this be `bytes` like all other protobuf specs?

The confusing thing is that this is called `rollup_id` but is actually an arbitrary rollup name that has to be hashed before getting the final `RollupId` (in this PR using`RollupId::from_unhashed_bytes`). On the other hand, the rest of the codebase refers to `RollupId` as a sequence of 32 bytes.

Reply:
I think we shouldn't use `from_unhashed_bytes` then. Let me see how I can make it just a sequence of 32 bytes and avoid using `from_unhashed_bytes`

Reply:
used bytes instead of string. that should be better.